src/default_parameters.jl

```diff
@@ -47,7 +46,10 @@ The default values of the keywords define the default model setup.
     # of the nlev vertical levels, defined by a generalised logistic function,
     # interpolating ECMWF's L31 configuration
     GLcoefs::GenLogisticCoefs = GenLogisticCoefs{NF}()
-    n_stratosphere_levels::Int=2        # number of vertical levels used for the stratosphere
+    n_stratosphere_levels::Int=2                                        # number of vertical levels used for the stratosphere
+    initialize_σ_manually::Bool=false                                   # initialize the σ vertical levels manually by providing a σ_levels_half 
+    σ_levels_half::Vector{Real}=Vector{Real}()                          # vector of σ half levels, only if set manually
+    nlev::Int=nlev_default(model, initialize_σ_manually, σ_levels_half) # number of vertical levels 
```

Comment:
I'd suggest to remove the `initialize_σ_manually` field, as 0-element vector for `σ_levels_half` could serve that purpose. See suggestion further down

Reply:
yep, I agree, that's more elegant